Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -49,6 +49,45 @@
     BOT_ID, tenant_id, service_url = BOT_ID.split('@')
     set_integration_context({'tenant_id': tenant_id, 'service_url': service_url})
 
+CLIENT_CREDENTIALS_FLOW = 'Client Credentials'
+AUTHORIZATION_CODE_FLOW = 'Authorization Code'
+AUTH_TYPE = PARAMS.get('auth_type', CLIENT_CREDENTIALS_FLOW)
+
+AUTH_CODE: str = PARAMS.get('auth_code', '')  # PARAMS.get('auth_code', {}).get('password', '')
+REDIRECT_URI: str = PARAMS.get('redirect_uri', '')
+SESSION_STATE = 'session_state'
+REFRESH_TOKEN = 'refresh_token'  # guardrails-disable-line
+
+CLIENT_CREDENTIALS = 'client_credentials'
+AUTHORIZATION_CODE = 'authorization_code'
+
+USER_LIST_HEADERS: dict = {
```

Comment:
The name for this global isn't understandable until you see how it's used.
Perhaps `USER_LIST_MARKDOWN_HEADERS`? 
Alternatively, `USER_LIST_HUMAN_READABLE_HEADERS`.